shared/src/mobileMain/kotlin/org/jetbrains/kotlinconf/ui/AboutConfScreen.kt

```diff
@@ -278,7 +267,7 @@ private fun LightningTalks() {
 
 @Composable
 private fun Party() {
-    AboutConfSubtitle("May 23, 9:00", "Party")
+    AboutConfSubtitle("May 23, 9:00", "KotlinConf’24 party ")
```

Comment:
Don't you want to extract it to the strings resources?

Reply:
thank you, fixed